### ISSUE TITLE
Not extracting Texture files

### ISSUE THREAD

**Issue by Findingo17** [mhvuze/MonsterHunterRiseModding#20]
For some reason why I try to use this, the models extract perfectly fine, but for some reason, the textures don't appear only the models' files do. I'm not a pro at understanding what the problem might be but once the full extracting is complete it says that some files were "skipped". Not sure what exactly the skipping means or is about precisely but I'm guessing that the skipped files are the texture ones. https://user-images.githubusercontent.com/109832922/180582914-80bc88a9-a216-4684-9811-5e4ceacdf88c.PNG

**Comment by Synthlight** [comment]
Files in the pak don't really have 'names', they have hashes, both for the name & ext.
When you extract you give it a file list with which it uses to translate the correct hashes into the right path/filename.
Not 100% of the files are on the list, and to prevent the unknown named files from being extracted, they are skipped by default.

You need to make sure you are:
- Using the up-to-date file list.
- Using the right file list for the version you have. (Steam/Swicth)
- DON'T HAVE ANY MODS INSTALLED. (From any manager that invalidates pak entries.)

Managers invalidate 'pak' path entries so the game thinks they aren't in there and looks in the external natives folder.
This of course means *anything* extracting from the pak won't see those same entries either.

So either you aren't using the correct/up-to-date list, or you have mods installed that hide those texture paths.

**Comment by Findingo17** [comment]
Seeing as it's just the demo I'm messing with I doubt any mods would affect it (plus I'm pretty sure I don't have mods but I might have to see if I did something odd with Nexus but Idk). As for the file list, I'm using the same one you get in the download it does say rise only and not sun break but I still get the DLC models just not the textures. But if to say that perhaps the current list im using isn't fully correct with what's in the .pak file, what can I do to update/fix it?

**Comment by Synthlight** [comment]
This is why I said "Using the right file list for the version you have".
![image](https://user-images.githubusercontent.com/1075166/180588234-cf095f3b-8ba5-434a-9f66-c01cabc2606a.png)

**Comment by Findingo17** [comment]
Ah ok, I'm still new to this so not only was I not fully aware of the multiple .list files, but of the fact that the demo works a bit different than the main game. I'll go have a look at those ones to see if it'll work.

**Comment by Synthlight** [comment]
I'm going to assume you got what you need with the correct list file. Re-op if still a issue.

**Comment by Findingo17** [comment]
Oh yeah sorry, It did work thank you for helping. But there is something else I'm wondering. I tried looking at an NPC model and noticed certain textures were still not present (eye texture for example) I looked in the list file for both the demo that I'm using as well as the full game one and saw that there doesn't seem to be any designation for and eye texture in the list file. Is there anything I can do like perhaps edit the file or have I missed something painfully obvious,
